src/azul/service/avro_pfb.py

```diff
@@ -329,6 +329,50 @@ def pfb_schema_from_replicas(replicas: Iterable[JSON]
     return keys, _avro_pfb_schema(values)
 
 
+def pfb_schema_from_anvil_schema(anvil_schema: JSON) -> tuple[Sequence[str], JSON]:
```

Comment:
L,
```suggestion
def pfb_schema_from_anvil_schema(anvil_schema: JSON
                                 ) -> tuple[Sequence[str], JSON]:
```
I do think it looks better the way you have it (which is barely over), but we do abide guidelines.